runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOverlay.java

```diff
@@ -90,17 +71,6 @@ public Dimension render(Graphics2D graphics)
 
 		panelComponent.getLines().clear();
 
-		if (MINING_ANIMATION_IDS.contains(client.getLocalPlayer().getAnimation()))
```

Comment:
I think this is worth trying to keep. Maybe add a timeout if there is a small window where you still mining with no animation.